src/app/progress-bar/progress-bar-example/progress-bar-example.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="card w-50 mt-2">
+  <div class="card-body">
+    <h4 class="card-title">Esempio barra di avanzamento</h4>
+
+    <p class="card-text" class="example-section">
+      <it-progress-bar [value]="valueEx1" [min]="minEx1" [max]="maxEx1"></it-progress-bar>
```

Comment:
In questo scenario potrebbe essere superfluo avere un binding per `min` e `max`. Sarebbe meglio inserire i valori direttamente nel file HTML.